api.swagger.yaml

```diff
@@ -6319,6 +6319,80 @@ paths:
           description: The user does not have access to the requested content/comment.
         '404':
           description: No content/comment with that UUID was found.
+  '/rate/{content_id}':
+    parameters:
+      - name: content_id
+        description: The Invotra UUID of content/comment.
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+          example: '01234567-89ab-cdef-1234-56789abcdef0'
+    put:
+      tags:
+        - rate
+      description: 'Rate a piece of content/comment.'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                rating:
+                  type: integer
+                  description: The rating for content/comment.
+                  example: 3
+                  minimum: 1
+                  maximum: 5
```

Comment:
I think we have level from 0 to 5.
So I am wondering if we need to set min as 0 instead of 1.
![image](https://user-images.githubusercontent.com/79103819/130810402-e099b77e-e201-412f-a7e2-02f1217ce6ec.png)